hack/ci-e2e-kind-gcs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2024 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -o errexit
+set -o nounset
+set -o pipefail
+
+make kind-up
+
+trap "
+  ( make kind-down )
+" EXIT
+
+kubectl wait --for=condition=ready node --all
+export KUBECONFIG=$KUBECONFIG_PATH
```

Comment:
Remove it, since we are setting here https://github.com/gardener/etcd-druid/blob/0e506d8a89e95fdcd7d8274730dc6f79d1ac8449/Makefile#L136

Reply:
This is removed in the latest commit that I pushed previously.